CONTRIBUTING.md

```diff
@@ -301,7 +301,9 @@ Stacked PRs (Y -> X -> `develop`) must never be merged until their base is final
 After merging the parent PR, child's base changes automatically (see [GitHub blogpost](https://github.blog/changelog/2020-05-19-pull-request-retargeting/))
 
 To create a release:
-* make sure [CHANGE-NOTES.html](CHANGE-NOTES.html) are updated
+* make sure the `Unreleased` section in [CHANGE-NOTES.md](CHANGE-NOTES.md) is updated
+* run `./gradlew patchChangeLog`, <br/>
```

Comment:
This change itself needs to be committed to `develop` in a PR (either a separate PR, or piggybacked on one of the PRs preceding the release), pls clarify